services/apps/alcs/src/providers/typeorm/migrations/1699461885734-add_outcome_public_search_noi.ts

```diff
@@ -0,0 +1,16 @@
+    }
+
+    public async down(queryRunner: QueryRunner): Promise<void> {
+        await queryRunner.query(`DELETE FROM "alcs"."typeorm_metadata" WHERE "type" = $1 AND "name" = $2 AND "schema" = $3`, ["VIEW","public_notice_of_intent_submission_search_view","alcs"]);
```

Comment:
The down here is only deleting the view, but not re-creating the previous version as it should. Did you delete too much?

Reply:
i just ran npm run alcs:typeorm migration:revert